packages/radix-vue/src/Menubar/MenubarMenu.vue

```diff
@@ -0,0 +1,75 @@
+<script lang="ts">
+import type { Ref, InjectionKey } from "vue";
+import type { DataOrientation } from "@/shared/types";
+import { useVModel } from "@vueuse/core";
+
+export interface MenubarMenuRootProps {
+  modelValue?: boolean;
```

Comment:
I think `MenubarMenu` doesn't required a modelValue, but instead a value.
If this value matches the MenubarRoot's  modelValue, then the menu will open.

reference: https://www.radix-ui.com/docs/primitives/components/menubar#menu